app/services/pandanet/client.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module Pandanet
+  # We connect to Pandanet to get information about an `Attendee`. We connect
```

Comment:
This must have taken a bit of work to investigate how the whole Pandanet/telnet thing worked -- the documentation you added is great.

Reply:
Thanks! It's a good thing that they mention the "client mode" thing in their docs, I never would have figured it out otherwise.